azure/durable_functions/models/ReplaySchema.py

```diff
@@ -6,3 +6,4 @@ class ReplaySchema(Enum):
 
     V1 = 0
     V2 = 1
+    V3 = 2
```

Comment:
Does the extension already support V3 schema?

Reply:
The WebJobs extension or the Python one? 
WebJobs does, yes, and it passes the correct value to upperSchemaVersionNew. 
This repo is adding support for V3 with this PR. See David's earlier comment about long polling HTTP - I think it is fine for us to detect schema V3 and support long timers without adding support for other V3 features like long polling, but feel free to correct me if I'm wrong or if you want me to do some other testing to be sure.

Reply:
I was asking about the WebJobs extension, making sure we didn't also require a new extension release (sounds like we don't)